garrysmod/gamemodes/terrortown/gamemode/lang/russian.lua

```diff
@@ -217,7 +217,7 @@ L.sb_deaths     = "Смертей"
 L.sb_score      = "Счёт"
 L.sb_karma      = "Карма"
 
-L.sb_info_help  = "Обыскав тело этого игрока, вы сможете ознакомиться с результами здесь."
+L.sb_info_help  = "Осмотрите тело этого игрока, чтобы пересмотреть результаты здесь."
```

Comment:
Pointless change due to a previous pointless change. This translation sounds worse than what was there before.

Reply:
My original localization for that string was odd. Yeah, it was almost word by word, but it was incorrect from the perspective of our grammar system. Anyway, it is grammatically correct and will be correct from a localization perspective if my suggestion for the original string will be accepted.
Additionally, "review" ≠ "ознакомиться" in this situation since we already seen the results when we were searching a 
dead body.